test5.js

```diff
@@ -0,0 +1,12 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( "hey" )
```

Comment:
Codacy has a fix for the issue: [Replace `·"hey"·)` with `"hey");`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
  console.log("hey");
```